src/overview/tabs/Settings/cards/SettingsCard.tsx

```diff
@@ -1,12 +1,13 @@
+/* eslint-disable max-lines-per-function */
```

Comment:
are u aimin to refactor this component ?

Reply:
yes